dap_job_quality/notebooks/job_quality_harder_dimensions.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
ditto:

`job_quality_harder_dimensions.ipynb`:
- i needed to pip install wordcloud
- on keyword searches - you could do some veeery minimal text cleaning (as defined in `text_cleaning.py` and lowercase the text so you can make more matches)

i.e. 
```
import dap_job_quality.utils.text_cleaning as tc

unlabelled_data['clean_description'] = unlabelled_data['description'].apply(tc.clean_text)
purpose_results = unlabelled_data[unlabelled_data['clean_description'].str.lower().str.contains('|'.join(phrases["reward"]))]
purpose_results.head()
```
and do that for all the other labels.